### ISSUE TITLE
Error on import: Undefined symbol hs_compile_lit_multi

### ISSUE THREAD

**Issue by RicoVZ** [darvid/python-hyperscan#28]
Since today, after installing hyperscan 0.1.5 on Python 3.6.9 (Ubuntu 18.04), importing hyperscan results
in an ImportError.

My Hyperscan package versions are:
* libhyperscan-dev/now 5.1.0-1 amd64 [installed,local]
* libhyperscan5/now 5.1.0-1 amd64 [installed,local]

The error:

```
import hyperscan

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/tmp/test3/lib/python3.6/site-packages/hyperscan/__init__.py", line 5, in <module>
    from hyperscan._hyperscan import *
ImportError: /tmp/test3/lib/python3.6/site-packages/hyperscan/_hyperscan.cpython-36m-x86_64-linux-gnu.so: undefined symbol: hs_compile_lit_multi

```

Occurs with `hyperscan-0.1.5-cp36-cp36m-manylinux2014_x86_64.whl`.
I noticed that this file was updated recently (11-12-2020), so I thought that might be the cause.

Does not occur with `hyperscan-0.1.5-cp36-cp36m-manylinux1_x86_64.whl`

Also does not occur with `hyperscan-0.1.5-cp38-cp38-manylinux2014_x86_64.whl` on a machine with Python 3.8.5 (Ubuntu 20.04).

Do you know of any things that might cause this?

**Comment by darvid** [comment]
I was transitioning CI for this project from semaphore to GitHub Actions, and it looks like it uploaded some new wheels to the existing ``v0.1.5`` release on PyPI when it shouldn't have. Should be fixed now, thanks for the heads up - ``v0.2.0`` has also been released now, which will require Hyperscan 5.2.

**Comment by darvid** [comment]
created a related issue (#29) for statically linking the C extension; if you have any questions or concerns about moving to a statically linked library please let me know, otherwise I think that would make more sense for the majority of use cases and allow for easy upgrades without requiring a rebuild of Hyperscan.
